recipes/rdkit_utilities/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rdkit_utilities" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rdkit_utilities-{{ version }}.tar.gz
+  sha256: b93c3de687785e7dc94fbfc8bb7b97b538d3e896c21e44ad7be1797190f68d37
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - numpy
+    - python >=3.7
+    - typing_extensions
+    - rdkit
```

Comment:
This is not listed upstream, but since you're upstream I assume that it is correct?

Reply:
Do you mean the pip package? Yeah, rdkit isn't on Pypi so it was difficult to list it as a requirement. There is an `rdkit-pypi` package if people do want to use it with pip, but it isn't maintained by the official source so I left it off.